spread.yaml

```diff
@@ -183,10 +177,10 @@ backends:
                   image: ubuntu-2004-64-virt-enabled
                   storage: 20G
                   workers: 7
-            - ubuntu-20.10-64:
-                  image: ubuntu-2010-64-virt-enabled
+            - ubuntu-21.04-64:
+                  image: ubuntu-2104-64-virt-enabled
                   storage: 20G
-                  workers: 6
+                  workers: 3
```

Comment:
Why just 3 workers here instead of 6 for 21.10 ?

Reply:
It is because we just run few tests on 21.04, I think it was an error to use 6 workers for 20.10

Reply:
For ubuntu-16.04-64 and ubuntu-18.04-64 we use 3 workers, we just use 6 for ubuntu-20.04-64 because we run many manual tests which are pretty slow

Reply:
Only 3 tests get run there. OTOH spread is smart enough to allocate just those 3 workers, so I suppose it's ok to leave it at 6 in case we add more tests and forget to bump the number.